### DISCUSSION TITLE
how to add super user and enable user-name password security in milvus lite

### DISCUSSION THREAD

**Discussion by davidzhr** [milvus-io/milvus#25269] [Q&A and General discussion]
hello everyone,

i use the below code to start the milvus.
`
config = MilvusServerConfig(proxy_port=19531, data_dir=r'd:\milvus_data')

server = MilvusServer(config=config)

server.start()

server.wait()
`

In MilvusServerConfig, i want to add another superuser and enable  password authentication, who have the best practice,  could you share it?

Thanks in advance.

**Comment by yhmo** [comment]
Read these doc to know how to configure multiple super users: 
https://milvus.io/docs/users_and_roles.md
https://milvus.io/docs/authenticate.md

Milvus creates a root user by default with a default password Milvus. The root user is granted the admin privileges, which means that this root user can have access to all resources and perform all actions.

By default, the `common.security.superUsers` field in the Milvus configuration file is empty, meaning that all users must provide the old password when resetting their password. However, you can designate specific users as super users who do not need to provide the old password. In the snippet below, root and foo are designated as super users.
```
common:
    security:
        superUsers: root, foo
```
